components/01-atoms/video-background/yds-video-background.twig

```diff
@@ -0,0 +1,39 @@
+{#
+ # Available variables:
+ # - video_background__content - the content of the video (typically an iframe)
+ #   the "|raw" filter is applied so that the iframe is rendered instead
+ #   of simply passing the iframe as plain text.
+ #
+ # Available blocks:
+ # - video_background__content - used to replace the content of the video with something other than the typical iframe
+ #   for example: to insert an html5 video component
+#}
+
+{% set video_background__base_class = video_background__base_class|default('video-background') %}
+{% set video_background__button__background_color = video_background__button__background_color|default('gray-800') %}
```

Comment:
Add this variable to the "Available Variables" comment above with the available options. I believe the "Callout" component has an example of this.